orc8r/gateway/python/magma/configuration/tests/BUILD.bazel

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@python_deps//:requirements.bzl", "requirement")
+load("//bazel:python_test.bzl", "pytest_test")
+
+MAGMA_ROOT = "../../../../../../"
+
+ORC8R_ROOT = "{}orc8r/gateway/python".format(MAGMA_ROOT)
+
+pytest_test(
+    name = "mconfig_manager_impl_tests",
+    size = "small",
+    srcs = ["mconfig_manager_impl_tests.py"],
+    imports = [ORC8R_ROOT],
+    deps = [
+        "//orc8r/gateway/python/magma/configuration:mconfig_managers",
+        requirement("protobuf"),
+    ],
+)
+
+pytest_test(
+    name = "mconfigs_tests",
```

Comment:
discussed in pairing - this test only needs mconfigs.py. This file should be modeled in a separate library.